src/modules/navigator/mission.cpp

```diff
@@ -237,6 +237,10 @@ Mission::update_offboard_mission()
 	bool failed = true;
 
 	if (orb_copy(ORB_ID(offboard_mission), _navigator->get_offboard_mission_sub(), &_offboard_mission) == OK) {
+		if (_offboard_mission.reset_mission) {
```

Comment:
Would prefer to trigger this on a vehicle status change with state == mission and disarmed.